### PR TITLE
Added celery + redis broker

### LINKED ISSUE #44: celery setup
**Is your feature request related to a problem? Please describe.**
Django because it is based on python has synchronous nature. In situations like sending emails, performing heavy tasks it is needed to be async. 

**Describe the solution you'd like**
Setup celery as a way to run tasks async with redis as the broker.

**Describe alternatives you've considered**
Another popular broker is rabbitmq + erlang. Redis is preferred as it also acts as a global cache + datastore whose functionalities can be proven to use later.

**Additional context**
Update requirements.txt and README.md file without fail.

### PR DESCRIPTION
**Pull request by sahil9001**
### Description
Added celery module for asynchronous tasks and redis-server as broker.

**Type of Change:** (Delete irrelevant options)
- Code

**Code/Quality Assurance Only** (Delete irrelevant options)

- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
No tests performed.

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by naveennvrgup — COMMENTED** [review]
Nice to see you follow our branch workflow :clap: and PR template. I would require these things to be added to this PR:
1.  Add a sample helloworld() so that we can test celery from `./manage.py shell`.
1. Add instructions to README.md on "how to use/start/stop this celery?"
    1. An even better option will be create a django app called 'core'. As django app are not strictly for MVP. It is also for isolating related functionalities. 
    1. Create a django management command  to start celery under app 'core'.

**Comment by sahil9001** [comment]
@naveennvrgup Sir, I've added the core app and running instructions for celery.

**Review by sumit-badsara — APPROVED** [review]
Good work with this @sahil9001 , any suggestions/improvements @naveennvrgup @PratibhaShrivastav ?

**Comment by sahil9001** [comment]
@naveennvrgup @PratibhaShrivastav Can you please provide reviews on this?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 1. Make a copy of `.env.sample` to `.env` and change the values of variables with original values.
 1. Migrate the migrations: `python manage.py migrate`
 1. Run the app: `python manage.py runserver`
+1. Open two terminals and run `celery -A server worker -l info` and `redis-server`  on second and third terminal respectively.
```
Comment (naveennvrgup) [inline README.md:11]: A brief of how to install redis will be helpful. I will put that as a good first issue for now. :)

**Review by naveennvrgup — COMMENTED** [review]
Great job on the code. Looks good on the surface

**Comment by naveennvrgup** [comment]
### I tired to test the async behaviour with help of `python3 ./manage.py shell` the test goes like this

image

### The celery is getting triggered thats a +ve sign :fire: . But when it throws this error. Please look into it :innocent: 

image
image

### Please go through the same testing procedure and ensuring the error is solved.

**Comment by sahil9001** [comment]
@naveennvrgup sir, I've resolved the timezone error.

**Review by naveennvrgup — APPROVED** [review]
1. Work as expected. Test locally
2. Merging the PR

**Pull request merged**